### ISSUE TITLE
Improper Naming for Examples

### ISSUE THREAD

**Issue by ManvendraJani** [JacquesCarette/Drasil#3521]
While working on #3491 I noticed that this the `progName` for `SglPendulum` and `DblPendulum` are odd. 
Single Pendulum
![image](https://github.com/JacquesCarette/Drasil/assets/58832908/4350ebb2-028a-4a4c-a1a3-59f08f332460)
Double Pendulum
![image](https://github.com/JacquesCarette/Drasil/assets/58832908/f38f5b13-d8bb-4f4e-800d-7915b9611e16)

There is a lot that looks wrong here,
- The UIDs are the same (I am not sure if this is ok in this situation as they are different examples)
- The proper nouns withing them (These are supposed to represent their full names) are the same
- The abbreviations are longer and contain more information than the name. 

I think this is a pretty simple fix but just wanted to make sure that this is not by design before changing this.

**Comment by samm82** [comment]
Good catches! Changing the UIDs, likely to "sglPendulum" and "dblPendulum", respectively, will likely be a unanimous decision.

However, changing the proper nouns to "Single Pendulum" and "Double Pendulum" may be more debated, since this change will likely have more consequences. I personally would do this, since it (a) differentiates the two and (b) allows us to throw them in the table of abbreviations!

**Comment by smiths** [comment]
I agree with @samm82.  The UIDs should be changed.  I also think the proper nouns should be changed.

I wonder if the abbreviations should be "SglPend" and "DblPend"?  I agree that there isn't much abbreviating in the current abbreviations.

I believe @cd155 did the most recent updates of Double Pendulum.  Any thoughts @cd155?

**Comment by cd155** [comment]
"SglPend" and "DblPend" sound good. Theoretically, we may combine them into one case study (Pendulum), but it is not very practical in this case (two cases use different solving methods). Single and double can belong to the family of the pendulum case study. We may also want to add a triple pendulum, so we may also call it a multiple pendulum.

**Comment by smiths** [comment]
@cd155 raises a good point about a family of pendulum models.  We know that we eventually want to be able to explicitly define families, but for now, we'll have to stick with implicitly defined families.
